app/views/items/_form.html.erb

```diff
@@ -27,7 +27,7 @@
               <% end %>
 
               <%= f.input :name, label: "Quantity Per Individual", wrapper: :input_group do %>
-                <%= f.input_field :distribution_quantity, class: "form-control" %>
+                <%= f.input_field :distribution_quantity, class: "form-control", value: f.object.distribution_quantity || (f.object.kit_id.present? ? 1 : 50) %>
```

Comment:
Why do we need this if we have an `after_initialize`?

Reply:
I thought it would be helpful for the user to see what the default was in the form. I just removed it and learned that we don't need it after all. Thank you for pointing this out.